docker-compose-prod.yml

```diff
@@ -0,0 +1,34 @@
+version: '2'
```

Comment:
Small nit: I think it should be docker-compose.prod.yml according to their style guide.